src/Devices/Ethernet_TC.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Ethernet_TC.h"
+using namespace std;
```

Comment:
Are you actually using anything from `namespace std` or is this just a habit? If you have anything from `std` then let's reference it directly as `std::`.